### ISSUE TITLE
Map a collection of DTOs to entities

### ISSUE THREAD

**Issue by furier** [leonardoporro/Detached-Mapper#76]
Id like to map a collection of DTOs to entities like this

```cs
var entities = await dbContext.MapManyAsync<Entity>(dtos);
```

Currently I have to do the following? which has some overhead with spawning a task for each DTO being mapped...

```cs
var entities = await Task.WhenAll(dtos.Select(dto => dbContext.MapAsync<Entity>(dto)));
```

**Comment by leonardoporro** [comment]
@furier Thanks for the feedback
Not only the tasks but also the DB queries. I'll improve it to load a bunch of entities and do the mapping. But there are some more urgent things I need to fix first.

**Comment by leonardoporro** [comment]
In v7.2 I changed the way how load and mapping were working and extracted the query outside of the  mapping process (it used to be inside an ugly RootEntityTypeMapper). 

Now I have free way to implement Map many. Will be available soon.

**Comment by leonardoporro** [comment]
Hey, it took time but map many is here... 
Not only the overhead of tasks were removed but also, all the entities are loaded in a single query so it goes to the DB once.

Take a look [here](https://github.com/leonardoporro/Detached-Mapper/blob/main/test/Detached.Mappers.EntityFramework.Tests/Features/MapMany.cs)

Please take a look and reopen if needed. All help/feedback/tests are welcome.

Sorry for the infinite delay but working for free not always fit with my schedule.

**Comment by leonardoporro** [comment]
Released in 8.0.6
